kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/rest/JbpmRestIntegrationTest.java

```diff
@@ -218,6 +229,22 @@ public void testBasicJbpmRequestManyAcceptHeaders() throws Exception {
 
     }
 
+    @Test
+    public void testBasicJbpmRequestAcceptHeadersStrictAndField() throws Exception {
+        assumeTrue(marshallingFormat == MarshallingFormat.JSON);
+
+        long processId = processClient.startProcess(CONTAINER_ID, PROCESS_ID_USERTASK);
+        ProcessInstance desc = processClient.getProcessInstance(CONTAINER_ID, processId);
+        TaskSummary[] tasks = desc.getActiveUserTasks().getTasks();
+        assertTrue(tasks.length > 0);
+
+        WebTarget target = newRequest(RestURI.build(TestConfig.getKieServerHttpUrl(), QUERY_URI + "/" + TASK_GET_URI, Collections.singletonMap(TASK_INSTANCE_ID, tasks[0].getId())));
+        Assertions.assertThat(target.request(MediaType.APPLICATION_JSON_TYPE).get(String.class)).contains("null");
+        Assertions.assertThat(target.request().header("content-type", MediaType.APPLICATION_JSON + ";fields=not_null;strict=true").get(String.class)).doesNotContain("null");
+        Assertions.assertThat(target.request().header("accept", MediaType.APPLICATION_JSON + ";fields=not_null;strict=true").get(String.class)).doesNotContain("null");
+        Assertions.assertThat(target.request().header("accept", MediaType.APPLICATION_JSON + ";fields=not_null").get(String.class)).doesNotContain("null");
```

Comment:
And just for complete completeness, you can add

`Assertions.assertThat(target.request().header("content-type", MediaType.APPLICATION_JSON + ";fields=not_null").get(String.class)).doesNotContain("null");`

Reply:
Added